tests/webhook_test.go

```diff
@@ -90,15 +91,38 @@ var _ = Describe("Validation webhook", func() {
 			)))
 		})
 
+		It("[test_id:TODO] should fail to create a second v1beta3.SSP CR", func() {
+			foundSsp := getSspV1Beta3()
+			ssp2 := foundSsp.DeepCopy()
+			ssp2.ObjectMeta = v1.ObjectMeta{
+				Name:      "test-ssp2",
+				Namespace: foundSsp.GetNamespace(),
+			}
+
+			err := apiClient.Create(ctx, ssp2, client.DryRunAll)
+			if err == nil {
+				Fail("Second SSP resource created.")
+				return
+			}
+			Expect(err.Error()).To(ContainSubstring(fmt.Sprintf(
```

Comment:
Use `MatchError`

Reply:
You can pass format strings to `ContainSubstring`. `fmt.Sprintf` is not required.

Reply:
done